src/main/resources/application.properties

```diff
@@ -2,14 +2,15 @@ logging.level.org.springframework.data=INFO
 # this is for sql scripts in our logs
 logging.level.org.springframework.jdbc.core.JdbcTemplate=DEBUG
 # this is for datasource
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
-spring.datasource.username=spark
-spring.datasource.password=123
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=jdbc:postgresql://localhost:5433/postgres
```

Comment:
why do you use `5433`?

Reply:
I use 5433, because I've got another DB on my localhost on port 5432.

Reply:
5433 - is a DB in a docker container

Reply:
I use a new DB in a docker container. I used another DB on port 5432 before.

Reply:
Please prefer using `6432`. This port is more often used for PG instead of `5432`